mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -87,13 +122,42 @@ function SiteInterventionsForm() {
         toast.error(language.error.submit)
       })
   }
+
+  const handleBiophysicalInterventionsOnChange = (event, intervention) => {
+    const biophysicalInterventionTypesCheckedCopy = biophysicalInterventionTypesChecked
+
+    if (event.target.checked) {
+      biophysicalInterventionsAppend({
+        interventionType: intervention,
+        interventionStartDate: '',
+        interventionEndDate: ''
+      })
+      biophysicalInterventionTypesCheckedCopy.push(intervention)
+    } else {
+      const fieldIndex = biophysicalInterventionsFields.findIndex(
+        (field) => field.interventionType === intervention
+      )
+      const typeIndex = biophysicalInterventionTypesCheckedCopy.findIndex(
+        (type) => type === intervention
+      )
+      biophysicalInterventionTypesCheckedCopy.splice(typeIndex, 1)
+      biophysicalInterventionsRemove(fieldIndex)
+    }
+    setBiophysicalInterventionTypesChecked(biophysicalInterventionTypesCheckedCopy)
+  }
+
+  const getBiophysicalIntervention = (intervention) =>
+    biophysicalInterventionsFields.find((field) => field.interventionType === intervention)
+
   return isLoading ? (
     <LoadingIndicator />
   ) : (
     <ContentWrapper>
       <FormPageHeader>
         <SectionFormTitle>Site Interventions</SectionFormTitle>
-        <Link to={-1}>&larr; {language.form.navigateBackToSiteOverview}</Link>
+        <Link to={`/sites/${siteId}/overview`}>
+          &larr; {language.form.navigateBackToSiteOverview}
```

Comment:
Whats `&larr;`? Whatever it is can we do `{'whatever'}` instead?. If it represents an ampersand for example, `{'&'}`

Reply:
i'm not sure why this appeared! will delete